test/test_type_promotion.py

```diff
@@ -174,38 +174,27 @@ def test_bfloat16(self, device):
             self.assertEqual((bf + scalar).dtype, torch.bfloat16)
             self.assertEqual((scalar + bf).dtype, torch.bfloat16)
             self.assertEqual(scalar + bf, bf + scalar)
-        with self.assertRaises(RuntimeError):
```

Comment:
can you add a complex scalar to the for loop above?  That seems to capture the intent of this test.

Reply:
Good idea. I also simplified the loop.